### PR TITLE
Help dialog 2

### PR DESCRIPTION
**Pull request by nkTribe**
This dialog is triggered by clicking the help button in the upper right hand corner of the graphs. They provide more information about the graph and the data being plotted. Look at HelpDialog/README for more information.

### CODE REVIEW

File: src/components/brands/HelpDialog/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+| `header2` | String | optional second header representing another topic to be explained
+| `dialogModel` | String | boolean to be associated with the dialog's `v-model`
+
+#### Note about `captions`
+If you want line breaks inside one caption, add `<br>` inside the caption string where you want the break to occur. The component will loop split the string and add in the <br> tag for you.
+
+## Computed Properties
+| Attribute | Type | Description
+| --- | --- | ---
+| `captionLineBreaks` | Array | array of strings representing caption split by where line breaks should occur. Line breaks are indicated by `<br>`
```
Comment (henryachen) [inline src/components/brands/HelpDialog/README.md]: To indicate an array of strings I would recommend using Array.<String> or String[]

Comment (nkTribe) [reply src/components/brands/HelpDialog/README.md]: But I did type Array?

File: src/components/brands/HelpDialog/HelpDialog.vue [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      max-width="389px">
+      <v-card>
+        <v-card-text>
+          <div
+            class="body-2">
+            {{header}}
+          </div>
+          <div
+            v-if="caption"
+            v-for="(line, index) in captionLineBreaks"
```
Comment (cdrappi) [inline src/components/brands/HelpDialog/HelpDialog.vue]: this is pretty gnarly

File: src/components/brands/HelpDialog/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Help Dialog
```
Comment (cdrappi) [inline src/components/brands/HelpDialog/README.md]: a README! this is really cool

**Pull request merged**